### PR TITLE
Improve blogs style

### PR DESCRIPTION
**Pull request by farellano0**
# Description
## What did I implement/Why did I implement this:

- **What?:** I've updated the filter styles and added new methods in application_helper, alongside enhancing the styling for Posts.
- **Why?:** These changes aim to make the filter options more intuitive and visually appealing, which enhances user experience. The addition of methods in application_helper is intended to streamline backend processes for better performance and maintenance.

## Screenshots
image
image

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,27 +1,48 @@
...
-    <div>
-      <%= link_to "Home", root_path, class: "inline-block rounded-lg text-center py-3 px-5 bg-blue-600 text-white block font-medium" %>
-      <%= link_to "New post", new_post_path, class: "inline-block rounded-lg py-3 px-5 bg-blue-600 text-white block font-medium" %>
-    </div>
+  <div class="flex justify-between items-center mb-4">
+    <h1 class="font-bold text-4xl m-0">Posts</h1>
   </div>
-
-  <div>
+  <div class="filtro mb-4">
```
Comment (MauricioNC) [inline app/views/posts/index.html.erb:10]: Change this class to English

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by PudinDeveloper — APPROVED** [review]
Don't forget to alphabetize the CSS classes

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -18,4 +18,10 @@ def self.convert_range_string_to_dates(dates_range)
   def navigation_text
     controller_name.singularize.humanize
   end
+
+  def truncate_words(text, length = 20, end_string = '...')
+    plain_text = text.to_plain_text
+    words = plain_text.split
+    words.length > length ? words[0...length].join(' ') + end_string : plain_text
+  end
```
Comment (juperameza) [inline app/helpers/application_helper.rb:26]: Doesn't this work 
https://apidock.com/rails/String/truncate

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -203,18 +204,19 @@ label {
...
 
 .custom-icon-circle {
   display: inline-block;
   background-color: white;
-  border-radius: 50%; 
-  padding: 8px; 
+  border-radius: 50%;
+  padding: 8px;
   box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.3);
   margin-left: 10px;
```
Comment (juperameza) [inline app/assets/stylesheets/application.css:215]: Order alphabetical

File: app/assets/stylesheets/filter.css [hunk 4]
```diff
@@ -2,6 +2,8 @@
   display: flex;
   flex-direction: column;
   flex-wrap: wrap;
+  background-color: #F3F4F6;
+  border-radius: 25px;
```
Comment (juperameza) [inline app/assets/stylesheets/filter.css:6]: Alphabetical order

File: app/assets/stylesheets/posts.css [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+.post {
+  border: 1px solid #ccc;
+  padding: 10px;
+  margin-bottom: 15px;
+  cursor: pointer;
+  border-radius: 5px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  transition: transform 0.3s ease, box-shadow 0.3s ease;
```
Comment (juperameza) [inline app/assets/stylesheets/posts.css:13]: Order this alphabetically

**Review by juperameza — COMMENTED** [review]
kudos on the helper
But order all your styles alphabetically 
And out of context
Stop filling roles in valorant stick to a role 🗣️🗣️

**Review by juperameza — CHANGES_REQUESTED** [review]
Lol didn't push on ask for changes

File: app/assets/stylesheets/application.css (hunk 3 above)
Comment (farellano0) [reply app/assets/stylesheets/application.css:215]: Done!

File: app/assets/stylesheets/posts.css (hunk 5 above)
Comment (farellano0) [reply app/assets/stylesheets/posts.css:13]: Done!

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (farellano0) [reply app/views/posts/index.html.erb:10]: Done!

Comment (MauricioNC) [reply app/views/posts/index.html.erb:10]: I'm still seeing the same class in Spanish

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (farellano0) [reply app/helpers/application_helper.rb:26]: Done!

File: app/assets/stylesheets/application.css (hunk 3 above)
Comment (farellano0) [reply app/assets/stylesheets/application.css:215]: Done!

File: app/assets/stylesheets/filter.css (hunk 4 above)
Comment (farellano0) [reply app/assets/stylesheets/filter.css:6]: Done!

**Review by juperameza — APPROVED** [review]

File: app/assets/stylesheets/home_page.css [hunk 6]
```diff
@@ -14,9 +14,10 @@
     justify-content: center;
 }
 
-.nav{
-    border-bottom: 5px solid grey;
+.degradado {
```
Comment (MauricioNC) [inline app/assets/stylesheets/home_page.css:17]: Change this class selector to English

File: app/assets/stylesheets/home_page.css [hunk 7]
```diff
@@ -1,8 +1,8 @@
-.carousel {
+.carrusel {
```
Comment (MauricioNC) [inline app/assets/stylesheets/home_page.css:1]: Also you still have this class selector in Spanish, change it to English

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by PudinDeveloper — APPROVED** [review]

File: app/assets/stylesheets/home_page.css (hunk 6 above)
Comment (farellano0) [reply app/assets/stylesheets/home_page.css]: Done!

File: app/assets/stylesheets/home_page.css (hunk 7 above)
Comment (farellano0) [reply app/assets/stylesheets/home_page.css]: Done!

**Review by MauricioNC — APPROVED** [review]

**Pull request merged**